sorc/grid_tools.fd/regional_esg_grid.fd/pmat5.f90

```diff
@@ -199,77 +195,81 @@ subroutine dctogr(xe,rlat,rlon)!                                       [ctogr]
 endif
 end subroutine dctogr
 
-!=============================================================================
+!> utility routine to calculate.
```

Comment:
Capitalize sentence.

Reply:
Replaced with ??? because it does not have documentation.